<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OSDesignProj3Class</class>
 <widget class="QMainWindow" name="OSDesignProj3Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1255</width>
    <height>988</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OSDesignProj3</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(240, 240, 240);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>10</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color: rgb(247, 247, 247);image: url(:/OSDesignProj3/left.png);}
QPushButton:hover{background-color: rgb(229, 243, 255);}
QPushButton:pressed{background-color: rgb(247, 247, 247);image: url(:/OSDesignProj3/left.png);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>11</width>
      <height>11</height>
     </size>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>971</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">gridline-color: rgb(13, 13, 13);
background-color: rgb(247, 247, 247);
selection-background-color: rgb(186, 186, 186);</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;\&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>10</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color: rgb(247, 247, 247);image: url(:/OSDesignProj3/search.png);}
QPushButton:hover{background-color: rgb(229, 243, 255);}
QPushButton:pressed{background-color: rgb(247, 247, 247);image: url(:/OSDesignProj3/search.png);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>961</width>
      <height>791</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
border:1px solid black;</string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>211</width>
      <height>791</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
border:1px solid black;</string>
    </property>
    <column>
     <property name="text">
      <string>目录结构</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>10</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color: rgb(247, 247, 247);image: url(:/OSDesignProj3/delete.png);}
QPushButton:hover{background-color: rgb(229, 243, 255);}
QPushButton:pressed{background-color: rgb(247, 247, 247);image: url(:/OSDesignProj3/delete.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1255</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="actionCreate_user"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCreate_user">
   <property name="text">
    <string>Create user</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="OSDesignProj3.qrc"/>
 </resources>
 <connections/>
</ui>
